Sources/Rendering/OpenGL/ImageMapper/index.js

```diff
@@ -37,6 +37,18 @@ function computeFnToString(property, pwfun, numberOfComponents) {
   return '0';
 }
 
+function splitStringOnEnter(inputString) {
+  // Split the input string into an array of lines based on "Enter" (newline) characters
+  const lines = inputString.split('\n');
+
+  // Remove any leading or trailing whitespace from each line and filter out empty lines
+  const trimmedLines = lines
+    .map((line) => line.trim())
+    .filter((line) => line.length > 0);
+
+  return trimmedLines;
```

Comment:
while this is nice to read, using split, map and filter is slow.
I think the following would be way faster:
```
const trimmedLines = [];

for (const line of inputString.split('\n')) {
  const trimmedLine = line.trim();

  if (trimmedLine.length > 0) {
    trimmedLines.push(trimmedLine);
  }
}

return trimmedLines;
```

Reply:
When I follow your recommendation, I encounter this issue. Perhaps you could consider turning off the eslint rule if you prefer this approach.

![CleanShot 2023-12-21 at 09 07 58](https://github.com/Kitware/vtk-js/assets/7490180/0e775dff-85c8-403e-8e57-fb5858815f0c)

Reply:
My bad, then let's do it simpler :

```
const lines = inputString.split('\n');
const trimmedLines = [];
for (let i = 0; i < lines.length; ++i) {
  const trimmedLine = lines[i].trim();
  if (trimmedLine.length > 0) {
    trimmedLines.push(trimmedLine);
  }
}
return trimmedLines;
```

Reply:
Done